16_wordle/main.go

```diff
@@ -78,24 +79,34 @@ func (l List) Random() []byte {
 }
 
 /*
-	Check every character of `input` and return character(s) that are in `chosenWord`
+Check every character of `input` and return character(s) that are in `chosenWord`
 
-	Convert at some point between byte and string type to return human readable output
+# Convert at some point between byte and string type to return human readable output
```

Comment:
Maybe now you can undo the conversion?

```suggestion
Convert at some point between byte and string type to return human readable output
```